src/World_TEST.cc

```diff
@@ -53,6 +53,17 @@ TEST(DOMWorld, Construction)
   EXPECT_FALSE(world.FrameNameExists("default"));
 
   EXPECT_EQ(1u, world.PhysicsCount());
+
+  auto errors = world.ValidateGraphs();
+  EXPECT_EQ(2u, errors.size());
+  EXPECT_EQ(errors[0].Code(), sdf::ErrorCode::FRAME_ATTACHED_TO_GRAPH_ERROR);
+  EXPECT_NE(std::string::npos,
+    errors[0].Message().find(
+      "FrameAttachedToGraph pointer is null"));
+  EXPECT_EQ(errors[1].Code(), sdf::ErrorCode::POSE_RELATIVE_TO_GRAPH_ERROR);
+  EXPECT_NE(std::string::npos,
+    errors[1].Message().find(
+      "PoseRelativeToGraph pointer is null"));
```

Comment:
Can we add some tests that have errors on the graphs themselves, not just missing graphs?

Reply:
a5985ab